### PR TITLE
Refactoring: Moving code shared by other projects to new modules in the extras package

### PR DESCRIPTION
**Pull request by GustavoHennig**

### CODE REVIEW

File: hpOneView/__init__.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 standard_library.install_aliases()
 
 __title__ = 'hpOneView'
-__version__ = '3.0.1'
+__version__ = '3.1.1'
```
Comment (marikrg) [inline hpOneView/__init__.py]: I believe the next release number will be 3.1.0
@tiagomtotti Do you agree?

Comment (tiagomtotti) [inline hpOneView/__init__.py]: Yep 3.1.0 !

Comment (GustavoHennig) [inline hpOneView/__init__.py]: Done

File: tests/unit/extras/test_mergers.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2016) Hewlett Packard Enterprise Development LP
```
Comment (marikrg) [inline tests/unit/extras/test_mergers.py]: Please change to 2016-2017

File: tests/unit/extras/test_comparators.py [hunk 3]
```diff
@@ -0,0 +1,456 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2016) Hewlett Packard Enterprise Development LP
```
Comment (marikrg) [inline tests/unit/extras/test_comparators.py]: Please change to 2016-2017

**Comment by marikrg** [comment]
Aprooved, looks good for me. I suggest merge this PR and https://github.com/HewlettPackard/oneview-ansible/pull/147 at the same time.

**Pull request merged**